dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -142,12 +145,13 @@ public fun DataFrame.Companion.readSqlTable(
     tableName: String,
     limit: Int = DEFAULT_LIMIT,
     inferNullability: Boolean = true,
+    dbType: DbType? = null,
```

Comment:
While having a dbType argument here is great for building and debugging support for new DbTypes, I do feel like it's not "complete custom support" of new datatypes.

To be fully extensible, like with `DataFrame.read()`, I think we should support having service-registered DbTypes. Then a user should be able to do:
```kts
implementation("org.jetbrains.kotlinx.dataframe...")
implementation("com.custom.dbtype-for-df")
```
and `DataFrame.readSqlTable(MyCustomDbConnection)` should work, recognizing the custom DBType that's on the classpath.

Still, it's just an idea and having this dbType argument here is still a must for when the automatic dbtype detection fails.